extensions/stats/plugin.cc

```diff
@@ -272,9 +272,14 @@ bool PluginRootContext::initializeDimensions(const json& j) {
         auto value =
             JsonGetField<std::string>(definition, "value").value_or("");
         if (name.empty() || value.empty()) {
-          return true;
+          LOG_WARN("empty name or value in  'definitions'");
```

Comment:
Want to make it crash in these two cases?

Reply:
Nope, changed to never return false. It'll just halt iteration but shouldn't break.

Reply:
Wait, I am confused.. Shouldn't it return true here in order to halt the iteration?

Reply:
The visitor call automatically halts iteration if any element returns false.

Reply:
Oh sorry, I thought the last commit is to remove `return false` from the second last commit. nvm.